contents/dataframes_missing.md

```diff
@@ -0,0 +1,196 @@
+## Missing Data {#sec:missing}
+
+Let's dive into how to **handle missing values** in `DataFrames.jl`.
+We'll cover **three main approaches for dealing with missing data**:
+
+1. **filtering missing** values with `ismissing` and `dropmissing`
+1. **filling or replacing missing** values with `coalesce`
+1. **skipping over missing** values with `skipmissing`
+
+First, we need a `DataFrame` filled  with missing values to showcase these approaches:
+
+```jl
+sco("""
+df_missing = DataFrame(;
+    name=[missing, "Sally", "Alice", "Hank"],
+    age=[17, missing, 20, 19],
+    grade_2020=[5.0, 1.0, missing, 4.0],
+)
+"""; process=without_caption_label)
+```
+
+This is the same `DataFrame` from @sec:dataframes but with some `missing` values added.
+
+### `Missing` type {#sec:missing_type}
+
+Some languages have several types to represent missing values.
+One such example is R which uses `NA`, `NA_integer_`, `NA_real_`, `NA_character_`, and `NA_complex_`.
+**Julia**, on the contrary, has **only one**: `Missing`.
+
+```jl
+sco("typeof(missing)"; process=without_caption_label)
+```
+
+As you can see `missing` is an instance of the type `Missing`.
+
+> **_NOTE:_**
+> In the [Julia Style Guide](https://docs.julialang.org/en/v1/manual/style-guide/),
+> there's a guidance to use camel case for types and modules
+> (see @sec:notation).
+
+The first thing we need to cover of the `missing` values is that they **propagate through several operations**.
+For example, addition, subtraction, multiplication, and division:
+
+```jl
+# addition # hide
```

Comment:
The comments don't add much value

Reply:
Then, make `Books.jl` render the correct block :)

Reply:
fair :rofl: